examples/knative-eventing-tutorial/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM golang:1-buster AS builder
```

Comment:
please unify the base images with @MBaczun. please use Alpine Linux ones.

Reply:
FROM amd64/golang:1.16.4-alpine3.13 as builder :)

Reply:
nope! use a custom `vhiveease/` tag for golang images (e.g., `vhiveease/vhive-golang:1.16` that is also `vhiveease/vhive-golang:latest` that is also `golang:1.16.4-alpine3.13`. then once we need to bump any version, the CI/CD pipeline should rebuild all images upon a change of the base image.

Mike, please look up the detailed instructions I gave in your PR.

Reply:
whoops! ok thanks

Reply:
same for the "run" image. use a custom tag image for it, like in the builder image's case. (use bare Linux Alpine for that, it has the smallest footprint)